crates/node-ethereum/tests/e2e/blobs.rs

```diff
@@ -85,7 +70,7 @@ async fn can_handle_blobs() -> eyre::Result<()> {
     // submit normal payload
     node.engine_api.submit_payload(payload, attributes, PayloadStatusEnum::Valid, vec![]).await?;
 
-    tokio::time::sleep(std::time::Duration::from_secs(3)).await;
+    tokio::time::sleep(std::time::Duration::from_secs(1)).await;
```

Comment:
can we use some of wait_until functions? e2e tests are already slow, hoping we can avoid sleeping for N seconds if possible